project/threads/templates/threads/base/links.html

```diff
@@ -10,14 +10,13 @@
 <link rel="shortcut icon" href="/favicon.ico">
 <meta name="msapplication-TileImage" content="/mstile-144x144.png">
 <meta name="msapplication-TileColor" content="#fcffff">
-
-<link type="text/css" rel="stylesheet" href="{% static "dependencies/materialize.min.css" %}">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```

Comment:
@AbhijithGanesh, develop branch includes Materialize CSS and JS files. Could you update your changes so that these files will be deleted?

Reply:
Yes, I shall update them

Reply:
Wait, in my branch those changes aren't there, you can view them here
[Here](https://github.com/AbhijithGanesh/OpenCiviWiki/blob/4c0388b7dd9de452fe5036b45d2b083d12809bd3/project/threads/templates/threads/base/links.html)

Reply:
Your branch does not have those files. Could you update your branch for these files?

Reply:
I'll fetch upstream

Reply:
I have added them now. The materialize css files are a part of it and now it is ready for merge I suppose @gorkemarslan

Reply:
Yes, they returned. :) Now you can delete them.

Reply:
Can you please clarify, I am not sure if I understand. Should I remove those two files?

Reply:
Yes, your new commit must have deletion operation of these files. After, merging we want `develop` branch not to have those files because our intent is to load them via CDN. So no need to have these files. 

This PR has two responsibilities:
1- Add CDN links to the templates
2- Delete existing materialize CSS and JS files

In this PR you only completed the first one. We want you to complete the second one, as well.

Before fetching upstream, you previously remove these files in your other branch and created a new branch from it So, you didn't have these files in your current branch. It is not possible to commit to deleting non-existing files. 

This is why I want to update your branch. First, have them and secondly delete them and commit this deleting operation.

Reply:
I shall delete those files

Reply:
I have made those changes @gorkemarslan